backend/djangoindia/api/urls/communication.py

```diff
@@ -1,9 +1,14 @@
 from django.urls import path
 
-from djangoindia.api.views.communication import ContactUsAPIView, SubscriberAPIView
+from djangoindia.api.views.communication import (
+    ContactUsAPIView,
+    SubscriberAPIView,
+    UnsubscribeAPIView,
+)
 
 
 urlpatterns = [
     path("subscriber/", SubscriberAPIView.as_view(), name="subscriber"),
+    path("unsubscribe/", UnsubscribeAPIView.as_view(), name="unsubscribe"),
```

Comment:
Since we are passing unsubscribe token in the url. we need to tell this to url.
```suggestion
    path("unsubscribe/<uuid:token>/", UnsubscribeAPIView.as_view(), name="unsubscribe"),
```